tests/interfaces/test_tf.py

```diff
@@ -528,6 +519,28 @@ def circuit_tf(phi, theta):
         assert np.allclose(autograd_grad[0], tf_grad[0], atol=tol, rtol=0)
         assert np.allclose(autograd_grad[1], tf_grad[1], atol=tol, rtol=0)
 
+    def test_qnode_jacobian_agrees(self, qnodes, tol):
+        """Tests that simple jacobian example is consistent."""
+        circuit, circuit_tf = qnodes
+
+        phi = [0.5, 0.1]
+        theta = [0.2]
+
+        phi_t = Variable(phi)
+        theta_t = Variable(theta)
+
+        jac = qml.grad(circuit, [0, 1])
+        autograd_jac = jac(phi, theta)
+
+        with tf.GradientTape() as g:
+            g.watch([phi_t, theta_t])
+            y = circuit_tf(phi_t, theta_t)
+
+        tf_jac = g.jacobian(y, [phi_t, theta_t])
+
+        assert np.allclose(autograd_jac[0], tf_jac[0], atol=tol, rtol=0)
+        assert np.allclose(autograd_jac[1], tf_jac[1], atol=tol, rtol=0)
```

Comment:
🥇

Reply:
Oh actually I remembered one thing I couldn't get to work here:
Line 532: `jac = qml.grad(circuit, [0, 1])`, this is `grad` and not `jacobian` which I couldn't get to work :thinking:
It seems ok since the qnodes in the test only have a 1D return.

The error I get with `jac = qml.jacobian(circuit, (0, 1))` is:
`TypeError: can only concatenate tuple (not "list") to tuple` (Although maybe this is something I'm doing)

Reply:
Oh, I think `autograd.jacobian` doesn't support multiple argnums when calculating the jacobian! This is a shortfall in autograd